### DISCUSSION TITLE
半夜不想睡觉 故发绳网求如何睡个好觉

### DISCUSSION THREAD

**Discussion by ShallowDream12321** [share121/inter-knot#1169] [其他]
委托奖励20菲林

**Comment by chaochao6667** [comment]
这个委托我接了

**Comment by kallen1314** [comment]
这我推荐去B站搜索一下助眠ASMR，我最喜欢的是benio店长的视频，还有很多这类的，比如goodnight moon、Alexandria、starling、yarify、cham、ppomo、judy、moonlight cottage，这都是国外做ASMR的。国内的我看的不多（听外语比听中文催眠多了），国内我看过的，B站有小奈别失眠和大猜这两个up主。助眠视频分类很多的，有情景扮演、触发、掏耳等。你可以去试试找找有没有你喜欢的

**Reply by ShallowDream12321** [reply]
这就去试试

**Comment by 10kaihua** [comment]
有菲林就是不一样[doge]

**Reply by ShallowDream12321** [reply]
必须的（叉腰）

**Comment by ran145s** [comment]
熬夜久了会失眠的，这是过来人的忠告。
不想英年早谢顶，就好好睡觉吧

**Reply by 10kaihua** [reply]
4分钟前，你也失眠了？🫢

**Reply by ShallowDream12321** [reply]
欧克 晚安好梦

**Comment by langxmo** [comment]
思考一下人生，没准能睡

**Reply by ShallowDream12321** [reply]
谢谢已睡

**Comment by ran145s** [comment]
我有躁郁症，失眠几年了，都习惯了，除了头发越来越少以外倒也没什么了



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年08月11日 02:31 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [share121/inter-knot] 半夜不想睡觉 故发绳网求如何睡个好觉 (Discussion #1169) |

思考一下人生，没准能睡

**Reply by ShallowDream12321** [reply]
抱抱你 祝你好梦

**Comment by lzxixizi** [comment]
分享多年来我用过最有效方法：直接熬夜到明天晚上，这样你就能直接睡着了

**Reply by ShallowDream12321** [reply]
听君一席话

**Comment by Kamisato100** [comment]
建议下楼跑十圈，包你回来沾床秒睡，睡不着再继续跑

**Reply by ShallowDream12321** [reply]
嘶，恐怖如斯

**Comment by T7moris** [comment]
这个委托我接了

**Reply by ShallowDream12321** [reply]
好呀

**Comment by Kiana-k423-Kaslana** [comment]
接委托的怎么这么快？开脚本了吧？

**Reply by ShallowDream12321** [reply]
哈哈哈哈

**Comment by su2569** [comment]
6

**Reply by ShallowDream12321** [reply]
9

**Comment by Fh52** [comment]
推荐飞机助眠法

**Reply by ShallowDream12321** [reply]
震惊

**Comment by ShallowDream12321** [comment]
祝大家好梦，身体健康

**Comment by xioagong** [comment]
太少了

**Comment by ShallowDream12321** [comment]
2024/8/12
